server/sonar-server/src/main/java/org/sonar/server/user/index/UserIndexer.java

```diff
@@ -20,6 +20,7 @@
 package org.sonar.server.user.index;
 
 import java.util.Iterator;
+import org.elasticsearch.action.index.IndexRequest;
 import org.elasticsearch.action.update.UpdateRequest;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.elasticsearch.action.update.UpdateRequest'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AUselessImportCheck)